packages/manager/modules/telecom-spare/src/index.js

```diff
@@ -0,0 +1,75 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-ovh-mondial-relay';
+import '@ovh-ux/ng-ovh-contracts';
+import '@ovh-ux/ng-ovh-simple-country-list';
+import '@ovh-ux/ng-ovh-telecom-universe-components';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import 'ovh-api-services';
+import 'oclazyload';
+import 'ovh-ui-angular';
+import 'angular-leaflet-directive';
+import 'angular-ui-bootstrap';
+import 'angular-translate-loader-partial';
+import 'angular-translate-loader-pluggable';
+import 'angular-translate-storage-cookie';
+import 'angular-translate-storage-local';
+
+import 'ovh-manager-webfont/dist/css/ovh-font.css';
+import 'ovh-ui-kit/dist/oui.css';
```

Comment:
This should be imported in the host app as it can involve stylesheet reset (cc @cbourgois @jleveugle )